.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# GitHub Dependabot configuration
+# Note: there is no interaction between this
+# configuration and dependabot security updates.
+# See here for more information:
+# https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/about-dependabot-security-updates#about-dependabot-security-updates
+
+version: 2
```

Comment:
Is this the latest version, or is there a reason for a specific version? Just curious :)

Reply:
Thanks, good question - this comes directly from examples and [GitHub's docs on the topic](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file):

`The file must start with version: 2`.